scripts/mvftoms.py

```diff
@@ -608,14 +598,12 @@ def _cp_index(a1, a2, pol):
                 s1 = time.time() - s
 
                 if average_data and utc_seconds.shape != ntime_av:
-                    print("Averaged %s x %s second dumps to %s x %s second dumps"
-                          % (np.shape(utc_seconds)[0], dataset.dump_period,
-                             ntime_av, dump_time_width))
+                    print(f'Averaged {np.shape(utc_seconds)[0]} x {dataset.dump_period} second dumps '
+                          f'to {ntime_av} x {dump_time_width} second dumps')
 
                 scan_size_mb = float(scan_size) / (1024**2)
 
-                print("Wrote scan data (%f MiB) in %f s (%f MiBps)\n"
-                      % (scan_size_mb, s1, scan_size_mb / s1))
+                print(f"Wrote scan data ({scan_size_mb} MiB) in {s1} s ({scan_size_mb / s1} MiBps)\n")
```

Comment:
The new format isn't using `:f`, which I think means it behaves like `%g`. That's probably fine (and maybe better) for this instance, but mentioning it in case you were wanting to keep the original behaviour.

Reply:
Thanks for pointing out - went for `:.3f` instead.